pages/stems/[id].tsx

```diff
@@ -149,6 +157,30 @@ const StemDetailsPage: NextPage<StemDetailsPageProps> = props => {
 						Sorry, no details were found for this stem.
 					</Typography>
 				)}
+
+				<Divider light sx={userProfileStyles.divider} />
+				<Typography variant="h4" gutterBottom>
+					Collaborations
+					<Typography component="span" sx={userProfileStyles.sectionCount}>
+						({projects && projects.length ? 0 : ''})
```

Comment:
This should show `(0)` when there are non instead of `()` in this case

<img width="1040" alt="image" src="https://user-images.githubusercontent.com/8240514/209274943-ddee146e-3079-4e65-89d2-00567a8ae5af.png">